src/test/java/org/jitsi/meet/test/pageobjects/web/SecurityDialog.java

```diff
@@ -246,4 +248,68 @@ private boolean getLockStateByClass(String className)
         WebDriver driver = participant.getDriver();
         return driver.findElements(By.className(className)).size() != 0;
     }
+
+    /**
+     * Returns the switch that can be used to detect lobby state or change lobby state.
+     * @return the lobby switch UI element.
+     */
+    private WebElement getLobbySwitch()
+    {
+        WebDriver driver = participant.getDriver();
+        WebElement lobbySection = driver.findElement(By.id(LOBBY_SECTION_ID));
+
+        if (lobbySection != null)
+        {
+            return lobbySection.findElement(By.tagName("input"));
+        }
+
+        return null;
+    }
+
+    /**
+     * Checks if the current conference has lobby enabled based on the security dialog's display state.
+     *
+     * @return {@code true} if the conference has lobby enabled in the security dialog, {@code false} otherwise.
+     */
+    public boolean isLobbyEnabled()
+    {
+        open();
+
+        WebElement lobbySwitch = getLobbySwitch();
+
+        return lobbySwitch != null ? lobbySwitch.isSelected() : false;
+    }
+
+    /**
+     * Toggles Lobby.
+     */
+    public void toggleLobby()
+    {
+        WebDriver driver = participant.getDriver();
+        WebElement lobbySwitch = getLobbySwitch();
+
+        if (lobbySwitch == null)
+        {
+            throw new NoSuchElementException("Lobby switch not found!");
```

Comment:
I thought `findElement` calls inside `getLobbySwitch()` would throw already

Reply:
Yep ... so probably I can remove this check ...